scripts/apps/search/components/actions-menu/Label.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {closeActionsMenu} from '../../helpers';
 
-export default function Label(props) {
+export const Label:React.StatelessComponent<any> = (props) => {
```

Comment:
not very important but can't we still `export default const Label...` ?

Reply:
That might be important, seems like I removed default export. I'll fix that on Thursday. Please keep an eye on similar situations. Most of the changes should be simply adding types and not changing any logic. There are a few exceptions, but it should be obvious there.

Reply:
Okay, also remember to take `import {Label}` back to `import Label`